std_msgs/Header header

map_creator/WsSphere[] WsSpheres

float32 resolution



